packages/node/src/indexer/worker/worker.ts

```diff
@@ -117,6 +130,8 @@ registerWorker({
   getStatus,
   syncRuntimeService,
   getSpecFromMap,
+  getMemoryLeft,
```

Comment:
We should be able to get this directly from the worker https://nodejs.org/api/worker_threads.html#workergetheapsnapshotoptions

Reply:
snapshots are huge, they could affect the performance.